src/components/impact/SDGBlock.js

```diff
@@ -211,10 +203,9 @@ const SDGBlock = () => {
       </Typography>
       <Grid container className={cardsContainer}>
         {sdgs.map(({ title, image, textOnOver, color }) => (
-          <Grid item className={cardContainer} key={title}>
+          <Grid item className={`${muiGridWithOnHoverText} ${cardContainer}`} key={title}>
```

Comment:
Just FYI, `className` can take an array, so you don't have to do string interpolation if you don't want to (not a problem to do it this way either though).

Reply:
Weird, I just tried and the console is complaining for an invalid type supplied

Reply:
Hrm, weird. There's a chance I'm thinking of the excellent [classnames](https://github.com/JedWatson/classnames) package - it definitely supports multiple classes, as well as some easy conditional logic.